src/ui/components/search/autocompletecomponent.js

```diff
@@ -192,10 +192,20 @@ export default class AutoCompleteComponent extends Component {
       sectionIndex: this._sectionIndex,
       resultIndex: this._resultIndex,
       promptHeader: this._originalQuery.length === 0 ? this.promptHeader : null,
-      listLabelIdName: this.listLabelIdName
+      listLabelIdName: this.listLabelIdName,
+      eventOptions: this.eventOptions(data)
     }));
   }
 
+  eventOptions (data) {
+    const eventOptions = data.sections?.map(section =>
```

Comment:
If `data.sections` is falsy and returns undefined, would the lookup in the HBS fail?

Reply:
the template HBS uses `{{#each sections}}` and `{{#each results}}` to drill into data.sections[index1].results[index2]. If data.sections is undefined, I believe it wouldn't call the lookup functions since it will not enter the two loops at all

Reply:
Got it. Even if it tried to lookup undefined data, it would simply return nothing